
Dataset device: cuda
TRAINING DATA: n=4908319
in_shape = torch.Size([34, 128, 128])
out_shape = torch.Size([10038])
Number of covariates = 34
Number of species = 10038
Dataset device: cuda
VALIDATION: n=85325
in_shape=torch.Size([34, 128, 128])
out_shape=torch.Size([10038]), n_species=10038
MODEL: cnn(
  (conv1): Conv2d(34, 32, kernel_size=(3, 3), stride=(1, 1))
  (act1): ReLU()
  (pool1): MaxPool2d(kernel_size=3, stride=3, padding=0, dilation=1, ceil_mode=False)
  (drop1): Dropout(p=0.3, inplace=False)
  (conv2): Conv2d(32, 8, kernel_size=(3, 3), stride=(1, 1))
  (act2): ReLU()
  (pool2): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
  (flat): Flatten(start_dim=1, end_dim=-1)
  (fc4): Linear(in_features=3200, out_features=10038, bias=True)
  (act4): ReLU()
)
EPOCH 0






































































































































































































































































































































































































































































  6%|██████▊                                                                                                           | 574/9587 [16:31<4:05:54,  1.64s/it]Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x7f5dffdd34c0>
Traceback (most recent call last):
  File "/home/nina/miniconda3/envs/glc23/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1479, in __del__
    self._shutdown_workers()
  File "/home/nina/miniconda3/envs/glc23/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1443, in _shutdown_workers
    w.join(timeout=_utils.MP_STATUS_CHECK_INTERVAL)
  File "/home/nina/miniconda3/envs/glc23/lib/python3.9/multiprocessing/process.py", line 149, in join
    res = self._popen.wait(timeout)
  File "/home/nina/miniconda3/envs/glc23/lib/python3.9/multiprocessing/popen_fork.py", line 40, in wait
    if not wait([self.sentinel], timeout):
  File "/home/nina/miniconda3/envs/glc23/lib/python3.9/multiprocessing/connection.py", line 931, in wait
    ready = selector.select(timeout)
  File "/home/nina/miniconda3/envs/glc23/lib/python3.9/selectors.py", line 416, in select
    fd_event_list = self._selector.poll(timeout)
KeyboardInterrupt:
  6%|██████▊                                                                                                           | 574/9587 [16:32<4:19:48,  1.73s/it]
Traceback (most recent call last):
  File "/home/nina/Documents/GeoLifeCLEF/simple_cnn.py", line 85, in <module>
    inputs = inputs.to(dev)
KeyboardInterrupt